bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
From what @Drup told me `version: dev` is required for opam to show the correct version when pinning. But I never noticed the difference myself.